wolox-training/igc-rails
# Kaminari

### Summary
- Added AJAX pagination using Kaminari

### Trello card
https://trello.com/c/z1BzVDGh/25-crear-lista-de-articulos-usando-api-rest

### Test plan
1. Go to the articles listing
2. Create articles until there's more than 5
3. A paginator should appear
4. Navigate through the pages using the paginator
5. Its content should get updated without reloading the page
